### DISCUSSION TITLE
If someone gives me an avatar id can I know which public key created it?

### DISCUSSION THREAD

**Discussion by javaspeak** [NextDotID/proof_server#102] [Q&A]
If someone gives me an avatar id can I know which public key created it?

**Comment by nykma** [comment]
What do you mean by Avatar ID? In our scenario, the material to distinguish different Avatars are the public keys of each avatar.

**Reply by javaspeak** [reply]
Am saying if I see the next.id avatar DID:

0x026ca64e70760e7635e10161095693abb08e9548ff84bed952fc08ce3ccc196736

is there a way of knowing the public key which created it?
